_posts/2022-08-05-crossplane-demo.md

```diff
@@ -0,0 +1,231 @@
+---
+layout:      posts
+background:  shortBackground
+title:       "Crossplane, a Kubernetes-powered IaC Tool"
+subtitle:    ""
+date:        2022-08-05 09:00:00
```

Comment:
Once this is approved, you'll want to update this date before merging.

Reply:
Updating this as I go along